examples/dataframe/app.py

```diff
@@ -93,12 +93,12 @@ def handle_edit():
     @render.text
     def detail():
         if (
```

Comment:
I like how you had it below in the test app. Copying it here.

```suggestion
        selected_rows = grid.input_selected_rows() or ()
        if len(selected_rows) > 0:
            # "split", "records", "index", "columns", "values", "table"
            return df.iloc[list(selected_rows)]
```